### PR TITLE
External SBG Systems device as alternative IMU

### PR DESCRIPTION
**Pull request by rousselmanu**
Application to communicate through UART with an SBG IG-500N device.
Gyrometer, accelerometer, magnetometer and barometer measurements are available, as well as quaternion/Euler angles/Rotation matrix, position and velocity estimate.
From now, only vehicle attitude topic is updated.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Thanks!

**Comment by LorenzMeier** [comment]
If you want to update on master please rebase on upstream/master (after using 'git fetch upstream') instead of merging. Thanks!

**Comment by rousselmanu** [comment]
Sorry, I was using the "sync" button on github software, which is not doing what I was expecting...

**Comment by LorenzMeier** [comment]
They use the Github flow, which is merging instead of rebasing. We're trying to avoid that, because it means that bisecting the history is virtually impossible.

**Comment by rousselmanu** [comment]
Ok, I understand. Thanks.

File: src/modules/sbg/sbgcom/comWrapper/comDataLog.c [hunk 1]
```diff
@@ -0,0 +1,203 @@
+#include "comWrapper.h"
```
Comment (LorenzMeier) [inline src/modules/sbg/sbgcom/comWrapper/comDataLog.c]: What is the license of this file?

**Comment by LorenzMeier** [comment]
@rousselmanu I need to have the license of the many files you checked in clarified to be able to accept this.

**Comment by LorenzMeier** [comment]
Closing this now as the author is not responding.

**Pull request closed without merging**